src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -1356,5 +1356,36 @@ void TwitchMessageBuilder::hostingSystemMessage(const QString &channelName,
                                MessageColor::System, FontStyle::ChatMediumBold)
         ->setLink({Link::UserInfo, channelName});
 }
+void TwitchMessageBuilder::deletionMessage(const MessagePtr originalMessage,
+                                           MessageBuilder *builder)
+{
+    builder->emplace<TimestampElement>();
+    builder->message().flags.set(MessageFlag::System);
+    builder->message().flags.set(MessageFlag::DoNotTriggerNotification);
+    builder->message().flags.set(MessageFlag::Timeout);
+    // TODO(mm2pl): If or when jumping to a single message gets implemented a link,
+    // add a link to the originalMessage
+    builder->emplace<TextElement>("A message from", MessageElementFlag::Text,
+                                  MessageColor::System);
+    builder
+        ->emplace<TextElement>(originalMessage->displayName,
+                               MessageElementFlag::Username,
+                               MessageColor::System, FontStyle::ChatMediumBold)
+        ->setLink({Link::UserInfo, originalMessage->loginName});
+    builder->emplace<TextElement>("was deleted: ", MessageElementFlag::Text,
```

Comment:
The suffix space here is extraneous and causes it to show up as a double space in Chatterino

![image](https://user-images.githubusercontent.com/962989/120920581-c3be1680-c6bf-11eb-87ab-782521bdb2fc.png)

```suggestion
    builder->emplace<TextElement>("was deleted:", MessageElementFlag::Text,
```